multimodal/src/autogluon/multimodal/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -251,3 +251,27 @@ model:
     data_types:
     additive_attention: False # Whether to use lightweight additive attention, can be True, False or "auto"
     share_qv_weights: False # Whether to share weight for query and value, can be True, False or "auto"
+  ft_transformer_refactor:
```

Comment:
Need to remove the configs of `categorical_transformer` and `numerical_transformer`?